Detections/SophosXGFirewall/ExcessiveAmountofDeniedConnectionsfromASingleSource.yaml

```diff
@@ -0,0 +1,26 @@
+id: 3d645a88-2724-41a7-adea-db74c439cf79
+name: Excessive Amount of Denied Connections from a Single Source
+description: |
+  'This creates an incident in the event that a single source IP address generates a excessive amount of denied connections.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: SophosXGFirewall
+    dataTypes: 
+      - Syslog
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Impact
+relevantTechniques:
+  - T1499
+query: |
+
+  let timeframe = ago(1h);
+  SophosXGFirewall
```

Comment:
This would need to be Syslog as that is the dataType this is flowing into

Reply:
These Analytics Rules require a Kusto Function that enables the SophosXGFirewall table to be used. Should the dataType be changes from Syslog to SophosXFirewall?

Reply:
Yes, if the table is not Syslog, then the name of the table is what you would use here. Should be the same as the dataTypes value.

Reply:
Corrections have been made and pushed.